src/Raven.Client/Documents/Smuggler/SmugglerResult.cs

```diff
@@ -24,6 +24,7 @@ public SmugglerResult()
             Tombstones = new CountsWithLastEtag();
             Identities = new Counts();
             Indexes = new Counts();
+            LegacyAttachments = new Counts();
```

Comment:
I don't think that we should have here `LegacyAttachments` as this is the statistics of the imported data, and we won't have in the UI legacy attachments stats. You convert the attachment and attach it to a document, so you should use the attachments stats inside the documents.

Reply:
sure